src/game.cpp

```diff
@@ -11584,29 +11583,33 @@ void game::chat()
         }
     }
 
-    if (available.empty()) {
-        add_msg(m_info, _("There's no-one close enough to talk to."));
-        if ( !query_yn(_("No one nearby. Yell?")) ) {
-            return;
-        }
-        u.shout();
-
-    } else if (available.size() == 1) {
+    if( available.size() == 1 ) {
```

Comment:
So, if there is exactly *one* NPC nearby, I will automatically talk to them and won't have the chance to yell. If there are several or none nearby, I can shout. Seems strange.

Btw. I see some code duplication: `talk_to_u(); moves -= 100; refresh_all();` is called here, and below after choosing an NPC. This should be in a function, or only at one place. If I wanted to change the moves based on what `talk_to_u` returns (the time of the talking), I would have to remember to change it at two separate places.